packages/clerk-js/turbo.json

```diff
@@ -22,6 +22,10 @@
         "!node_modules/**"
       ]
     },
+    "build:sandbox": {
+      "dependsOn": ["^build"],
+      "inputs": ["sandbox/**"]
```

Comment:
We will also want to configure `outputs` to cache assets produced by this new task